### ISSUE TITLE
CMV colitis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6408]
**Preferred term label:**
CMV colitis

**Synonyms**
Cytomegalovirus colitis

**Definition (free text, please give PubMed ID)**
Colitis, in the presence of a CMV infection.
PMID: 31194388

**Parent term (use hpo.jax.org/app)**
Colitis

**Comment by pnrobinson** [comment]
added
